builtin/bisect--helper.c

```diff
@@ -421,6 +638,8 @@ int cmd_bisect__helper(int argc, const char **argv, const char *prefix)
 			 N_("check whether bad or good terms exist"), BISECT_NEXT_CHECK),
 		OPT_CMDMODE(0, "bisect-terms", &cmdmode,
 			 N_("print out the bisect terms"), BISECT_TERMS),
+		OPT_CMDMODE(0, "bisect-start", &cmdmode,
+			 N_("start the bisect session"), BISECT_START),
```

Comment:
Looking at the range-diff, I noticed that the corresponding usage line was dropped, probably inadvertently. The original commit has it: https://github.com/gitgitgadget/git/commit/e14e913797bbd361a0929861699d983cabc430c8#diff-f42cbfc70e72a333b6ef20f77bf50af0R30

Could you add that back, please?

Reply:
It seems that `bisect-terms` was also dropped. I'll add it too.
https://github.com/pranitbauva1997/git/commit/8e53702074990ef6237ded020abf4d04ef5dc859#diff-f42cbfc70e72a333b6ef20f77bf50af0R26